crmsh/cibconfig.py

```diff
@@ -816,26 +816,55 @@ def postprocess_cli(node, oldnode=None, id_hint=None):
     resolve_references(node)
     if oldnode is not None:
         remove_id_used_attributes(oldnode)
+    if complete_advised:
+        complete_advised_meta(node)
     return node, obj_type, obj_id
 
 
+def complete_advised_meta(node):
+    """
+    Complete advised meta attributes
+    """
+    if node.tag != "clone":
+        return
+    primitive_list = node.xpath('primitive')
+    if not primitive_list:
+        return
+    set_list = []
+    for meta_item in ["promotable", "interleave"]:
+        if not is_attr_set(node, meta_item):
+            set_list.append(meta_item)
```

Comment:
IIUC we could just generally add `interleave=true` as long as it's not set here, no matter if it's promotable clone or not?